.editorconfig

```diff
@@ -1,13 +0,0 @@
-root = true
-
-[*]
-end_of_line = lf
-insert_final_newline = true
-indent_style = space
-indent_size = 2
-charset = utf-8
-trim_trailing_whitespace = true
-
-[*.go]
-indent_style = tab
-indent_size = 8
```

Comment:
Why has this file been removed?

Reply:
I originally forked this repo to update the code to what we needed it to do. To make it standard with other go projects, we just use gofmt and let it take care of this type of thing. Go standard is to use 4 spaces.

Reply:
That must have changed.  When i initially built this library, this was the standard.

Reply:
Actually, I think you're right that they use tabs, but it is normally 4 indent_size. However, the point remains that allowing gofmt to sort out those details standardizes the code.

Reply:
> Indentation
>     We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.

https://golang.org/doc/effective_go.html#formatting

Not sure where the indent_size of 8 went, but it was that a while ago

Reply:
What I'm saying is that because gofmt will re-indent the file to use tabs, setting this setting in editorconfig may just adjust the _visual_ width of the tab, but not the actual width in the file. So I'm not sure that having those settings there actually will change anything. Happy to add it back if you'd like.